packages/theme-patternfly-org/pages/global-css-variables.md

```diff
@@ -42,6 +55,24 @@ Where...
 - `breakpoint` is a media query breakpoint such as `sm` for `$pf-global--breakpoint--xs`.
 - `pseudo-element` is either `before` or `after`.
 
+Examples:
+- Note component variables are scoped to the top-level component selector
+  ```css
+  .pf-c-button {
+    /* Default, primary, and primary hovered button background colors */
+    --pf-c-button--BackgroundColor: transparent;
+    --pf-c-button--m-primary--BackgroundColor: var(--pf-global--primary-color--100);
+    --pf-c-button--m-primary--hover--BackgroundColor: var(--pf-global--primary-color--200);
+  }
+  ```
+
+<br/>
+Component variables can be found listed at the bottom of each component page (ex: [button css variables](/components/button#css-variables)).
```

Comment:
I believe we use `i.e.,` for things like this, or I do anyways :)

```suggestion
Component variables can be found listed at the bottom of each component page (i.e., [button css variables](/components/button#css-variables)).
```

Reply:
Actually, in our writing standards, the recommendation is to just write out "for example," See https://www.patternfly.org/v4/ux-writing/writing-design-guidelines-for-patternfly#style-voice-and-tone